docs/package.json

```diff
@@ -93,7 +93,7 @@
     "@babel/plugin-transform-react-constant-elements": "^7.23.3",
     "@babel/preset-typescript": "^7.23.3",
     "@mui-internal/docs-utils": "^1.0.0",
-    "@mui-internal/typescript-to-proptypes": "^1.0.2",
+    "@mui-internal/typescript-to-proptypes": "https://pkg.csb.dev/mui/material-ui/commit/fd499021/@mui-internal/typescript-to-proptypes",
```

Comment:
Friendly reminder to merge and release the relevant package before merging this PR. 😉